probe-rs/src/debug/debug_info.rs

```diff
@@ -560,7 +562,19 @@ impl DebugInfo {
                     tracing::trace!(
                         "UNWIND: No exception context found. Stack unwind will continue."
                     );
-                    None
+                    // Check LR values to determine if we can continue unwinding.
+                    if unwind_registers
```

Comment:
Shouldn't we find the actual entry point? With the esp-idf image format, the reset vector is not at 0, but somewhere in the middle of the address space.

Reply:
I'm not sure I follow you. This code relies on the debug info containing valid rules for unwinding the registers. Are you suggesting we add vendor specific logic to infer the entry point, instead of ending the unwind when the debug info is insufficient?

Reply:
Never mind, I think I'm just wrong